CDP4Composition/Composition/COMETBootstrapper.cs

```diff
@@ -126,8 +126,15 @@ private void AddPluginCatalogs(AggregateCatalog catalog)
 
             foreach (var directoryCatalog in pluginLoader.DirectoryCatalogues)
             {
-                catalog.Catalogs.Add(directoryCatalog);
-                UpdateBootstrapperStatus($"DirectoryCatalogue {directoryCatalog.FullPath} Loaded");
+                try
+                {
+                    catalog.Catalogs.Add(directoryCatalog);
+                    this.UpdateBootstrapperStatus($"DirectoryCatalogue {directoryCatalog.FullPath} Loaded");
+                }
+                catch (Exception ex)
```

Comment:
I dont think this is required. The exception is already handled in app.xaml.cs. This risks hiding an error we might want to be visible.

Reply:
Yes, well spotted. This is something we do every time (temporary handler), because errors are visible in the LoaderExceptions property. Forgot to remove it this time.
I'll refactor it to throw a new error that contains info from the LoaderExceptions that contains more useful information.